concourse/pipeline.yml

```diff
@@ -126,6 +128,8 @@ templates:
       RESPONSE_OPERATIONS_UI_PORT: '80'
       SAMPLE_SERVICE_HOST: rm-sample-service-ci.apps.devtest.onsclofo.uk
       SAMPLE_SERVICE_PORT: '80'
+      SAMPLE_HOST: rm-sample-service-ci.apps.devtest.onsclofo.uk
```

Comment:
Should we not be adding SAMPLE_URL? https://github.com/ONSdigital/response-operations-ui/pull/160/files#diff-fdf313ed2303059c44a9ebfffbef8eebR47